app/views/placements/schools/partner_providers/remove.html.erb

```diff
@@ -9,15 +9,27 @@
     <div class="govuk-grid-column-two-thirds">
       <label class="govuk-label govuk-label--l">
         <span class="govuk-caption-l"><%= @partner_provider.name %></span>
-        <%= t(".are_you_sure") %>
+        <%= t(".are_you_sure", provider_name: @partner_provider.name) %>
       </label>
+
+      <%= simple_format(t(".you_will_no_longer")) %>
+
+      <% if @partner_provider.becomes(Placements::Provider).placements.where(school: @school).exists? %>
+      <p class="govuk-body">Placements with this provider are:</p>
+      <ul class="govuk-list govuk-list--bullet">
+        <% @partner_provider.becomes(Placements::Provider).placements.where(school: @school).find_each do |placement| %>
```

Comment:
You're calling:
`@partner_provider.becomes(Placements::Provider).placements.where(school: @school)` twice.
It would be good to put this somewhere as a memoized method, possibly a decorator.